src/Module/Profile.php

```diff
@@ -350,7 +350,7 @@ public static function content(array $parameters = [], $update = 0)
 		$items = DBA::toArray($items_stmt);
 
 		if ($pager->getStart() == 0 && !empty($a->profile['profile_uid'])) {
-			$pinned_items = Item::selectPinned($a->profile['profile_uid'], ['uri']);
+			$pinned_items = Item::selectPinned($a->profile['profile_uid'], ['uri', 'pinned'], ['true' . $sql_extra]);
```

Comment:
Is the `true` necessary?

Reply:
Yeah. The variable `$sql_extra` does contain stuff like ` AND NOT private`. So we have to work around that `AND`.